app/views/pensions/update/duplicateSubmissionWarning.scala.html

```diff
@@ -0,0 +1,69 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import includes.forms.submit
+@import uk.gov.hmrc.tai.forms.YesNoForm
+@import includes.{cancelLink, ptaHeader}
+@import uk.gov.hmrc.play.views.html.helpers.{errorSummary, form}
+@import includes.forms.inputRadioGroupStyling
+
```

Comment:
Is this the same view as the employment journey warning?

Reply:
They are different views but they now share a partial